server/src/pages/settings/controller.js

```diff
@@ -0,0 +1,33 @@
+/* browser: true */
+
+const page = require("./page").page;
+
+let model;
+
+exports.launch = function(m) {
+  model = m;
+  render();
+};
+
+function render() {
+  page.render(model);
+}
+
+exports.disconnectDevice = function() {
+  let url = model.backend + "/api/disconnect-device";
+  let req = new XMLHttpRequest();
+  req.open("POST", url);
+  req.setRequestHeader("content-type", "application/x-www-form-urlencoded");
+  req.onload = function() {
+    if (req.status >= 300) {
+      // FIXME: a lame way to do an error message
+      window.alert("Error disconnecting: " + req.status + " " + req.statusText);
+    } else {
+      location.href = model.backend + "/settings";
+    }
+  };
+  console.log("csrf: " + model.csrfToken);
```

Comment:
You should do an eslint check, I'm guessing it will fail, as we have a console.log rule.